redux/auth/index.ts

```diff
@@ -0,0 +1,6 @@
+export { login, logout } from './actions';
+export { LOGIN, LOGOUT, RequestStatus as LoginRequestStatus } from './types';
+
+export { default } from './reducer';
+
+export { default as authSaga } from './saga';
```

Comment:
How is it possible you export that modules without importing before ? :thinking:

Reply:
I think it's possible since ES6 (or ES6++) :D